src/analytics.ts

```diff
@@ -30,41 +31,26 @@ export function createAnalyticsInstance(options?: Options) {
         growthbookOptions,
         disableRudderstackAMD = false,
     }: Options) => {
-        let CloudflareCountry = ''
-        try {
-            const response = await fetch('https://www.cloudflare.com/cdn-cgi/trace')
-
-            if (response.ok) {
-                const text = await response?.text()
-                const entries = Object.fromEntries(text.split('\n').map(v => v.split('=', 2)))
-
-                if (entries.loc) {
-                    CloudflareCountry = entries.loc.toLowerCase()
-                } else {
-                    console.warn('Location not found in the response.')
-                }
-            }
-        } catch (error) {
-            console.warn('Cannot get the Cloudflare location:', error)
-        }
+        const countryFromCloudflare = await CountryUtils.getCountry()
```

Comment:
this is the getCountry method:
https://github.com/deriv-com/deriv-utils/blob/master/src/utils/country.utils.ts